src/pybind/mgr/dashboard/frontend/src/app/shared/forms/cd-validators.ts

```diff
@@ -238,4 +228,24 @@ export class CdValidators {
       );
     };
   }
+
+  /**
+   * Validator function for UUIDs.
+   * @param required - Defines if it is mandatory to fill in the UUID
+   * @return Validator function that returns an error object containing `invalidUuid` if the
+   * validation failed, `null` otherwise.
+   */
+  static uuid(required = false): ValidatorFn {
+    const uuidRe = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
```

Comment:
Similar to the test suggestion, only here with `[1-5]`.

Reply:
I think your suggestion is actually less readable than a simple regular expression.

Reply:
I agree that my solution does not provide a much better understanding but you only need one place to define a regex for hexadecimal numbers as it is redefined 5 times through the regex creation. At least through the insensitive option the regex definition is pretty clear.  So I'm torn between those two solutions, I can live with both but both aren't a perfect match, even so they return one.

I will resolve this conversation.